repos/fdbt-site/tests/pages/api/searchOperators.test.ts

```diff
@@ -61,22 +61,22 @@ describe('searchOperators', () => {
         });
     });
 
-    it('should redirect to itself (i.e. /searchOperators) with search text in the query string when the search term is valid', () => {
+    it('should redirect to itself (i.e. /searchOperators) with search text in the query string when the search term is valid', async () => {
         const { req, res } = getMockRequestAndResponse({
             body: {
                 search: 'Search',
                 searchText: 'manchester',
             },
         });
 
```

Comment:
We need a test in this file to simulate an operator being selected which has no services, and check to see the API behaves correctly.